### ISSUE TITLE
How to use the JS version instead of the Java version?

### ISSUE THREAD

**Issue by nebrelbug** [ampproject/rollup-plugin-closure-compiler#158]
When I add this as a plugin, it throws an error about Java not being in the path. How can I make the plugin use the JS version of Closure Compiler?

Thanks!

**Comment by kristoferbaxter** [comment]
Interesting, by default this plugin attempts to use the `Native` platform, falling back to `Java`, then `JavaScript`.

```javascript
function getNativeImagePath() {
  if (process.platform === 'darwin') {
    try {
      return require('google-closure-compiler-osx');
    } catch (e) {
      return;
    }
  }
  try {
    return require('google-closure-compiler-linux');
  } catch (e) {
  }
}
```

My assumption is this is a Windows machine?

I'll look at providing an option to force a specific platform.

**Comment by kristoferbaxter** [comment]
If someone wants to tackle this, the right place to look is `compiler.ts`, inspect the `compileOptions` and use the data to set the value passed to `getFirstSupportedPlatform`.

**Comment by nebrelbug** [comment]
@kristoferbaxter I would be interested in working on this.

**Comment by nebrelbug** [comment]
Hi @kristoferbaxter! Any progress?

**Comment by kristoferbaxter** [comment]
Resolved with `0.12.0`.
